subsys/net/lib/nrf_cloud/src/nrf_cloud_fota.c

```diff
@@ -104,6 +106,31 @@ static bool reboot_on_init;
  */
 static bool fota_report_ack_pending;
 
+static void update_last_job(char const *const job_id)
+{
+	if ((job_id == NULL) || (job_id[0] == '\0')) {
+		return;
+	}
+
+	K_SPINLOCK(&last_job_slock) {
+		strncpy(last_job, job_id, sizeof(last_job) - 1);
+		last_job[sizeof(last_job) - 1] = '\0';
+	}
+
+	LOG_DBG("Updated last job: %s", job_id);
+}
+
+static bool is_last_job(char const *const job_id)
+{
+	bool match;
+
+	K_SPINLOCK(&last_job_slock) {
+		match = (strncmp(last_job, job_id, sizeof(last_job)) == 0);
```

Comment:
This will return false if a job_id which was longer than `NRF_CLOUD_FOTA_JOB_ID_SIZE` was stored, even if the same job_id was called

I don't have a problem with that, just want to make sure its the intended behavior